django_jalali/db/models.py

```diff
@@ -318,8 +318,7 @@ def value_to_string(self, obj):
         if val is None:
             data = ''
         else:
-            d = datetime_safe.new_datetime(val)
-            data = d.strftime('%Y-%m-%d %H:%M:%S')
+            data = str(val)
```

Comment:
@MrVIII Also it's better to use @python_2_unicode_compatible for this class to avoid python 2 unicode errors as Django doucmentation suggests.